### PR TITLE
docs: update per-route serializer example

### PR DESCRIPTION
**Pull request by climba03003**
Resolve: #32 

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,19 +41,11 @@ fastify.register(require('fastify-accepts-serializer'), {
...
-  }
-}
-
-fastify.get('/request', { config }, function (req, reply) {
-  reply.send({pippo: 'pluto'})
+fastify.get('/request', function (req, reply) {
+  reply
+    .type('application/x-protobug')
+    .serializer((body) => AwesomeMessage.encode(AwesomeMessage.create(body)).finish())
+    .send({pippo: 'pluto'})
```
Comment (jsumners) [inline README.md:48]: These are vastly different implementations. One is reusable across multiple routes (the original configuration based approach) and the other requires repetitions across multiple route handler implementations.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-accepts-serializer",
-  "version": "3.0.1",
+  "version": "3.1.0",
```
Comment (jsumners) [inline package.json:3]: Do not modify the package version in PRs.

Comment (climba03003) [reply package.json:3]: actually, it is merge from upstream. not modify by me.

File: README.md (hunk 1 above)
Comment (climba03003) [reply README.md:48]: The route level implementation is copy from test file. If you want it to be shared across other route. I can modify it and extract it to be a variables.

File: package.json (hunk 2 above)
Comment (jsumners) [reply package.json:3]: `git checkout master && git pull && git checkout your-branch && git rebase master`

Comment (climba03003) [reply package.json:3]: A little bit late, I already removed the repo.... I will reopen another one. 
Thanks for your suggestion, I will use it another time.

**Comment by climba03003** [comment]
The git commit history is a little bit mess in here and I have already removed the original repo in my account. So, I will close this one and re-create the new one. Sorry to messing up here.

**Pull request closed without merging**